nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -415,6 +415,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - `retroArchCores` has been removed. This means that using `nixpkgs.config.retroarch` to customize RetroArch cores is not supported anymore. Instead, use package overrides, for example: `retroarch.override { cores = with libretro; [ citra snes9x ]; };`. Also, `retroarchFull` derivation is available for those who want to have all RetroArch cores available.
 
+- `/usr` will always be mounted in the initial ramdisk. See the `fileSystems.<name>.neededForBoot` option.
```

Comment:
This doesn't provide much context on the implications. From https://github.com/NixOS/nixpkgs/pull/146142#issuecomment-974735181:

> well, it would. Yet, adding a path to the defaults of neededForBoot also means its inclusion in the initrd image, I suppose. Now, in NixOS, by default there's only a single link there, but some user might have placed more files there. Upgrading and not knowing of this change could result in a larger initrd for that user.

Can you add some of this into here, so people reading the release notes know if they're affected by this, and how?

Reply:
Some explanation added.